### PR TITLE
Add replayer for capture files

### PR DESCRIPTION
**Pull request by sandermvanvliet**
This PR adds a command line utility that you can point at a PCAP file in order to easily debug messages that were captured during a Zwift session.

Should save a lot of sweat trying to analyse or reproduce issues ;-)

### CODE REVIEW

File: ZwiftPacketMonitor.Replay/DependencyRegistration.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        public static ServiceProvider Register()
+        {
+            var serviceCollection = new ServiceCollection();
+
+            // added to allow logging level configuration, change MinLevel to LogLevel.Debug to see internal traces.
+            serviceCollection.AddLogging(configure => configure.AddConsole())
+                .Configure<LoggerFilterOptions>(configure => configure.MinLevel = LogLevel.Information);
+
+            serviceCollection.AddZwiftPacketMonitoring();
+            serviceCollection.AddSingleton<Replay>();
```
Comment (braddwalker) [inline ZwiftPacketMonitor.Replay/DependencyRegistration.cs:17]: I'm getting build errors here. Shouldn't this be `Replayer` and not `Replay`, given that the former is a type name, and the latter is a namespace?

**Review by braddwalker — CHANGES_REQUESTED** [review]
Overall I like these changes. There's one issue with the DI registration that I'd like you to look at before I can merge.

**Comment by braddwalker** [comment]
Was you intention to abandon this PR? It seems like you re-worked some of these changes into #29

**Comment by sandermvanvliet** [comment]
Yeah I think it’s better to close this one. Sorry I didn’t think about that sooner!

**Pull request closed without merging**